src/relay/arweave_meta_bundler.go

```diff
@@ -120,13 +111,21 @@ func (self *ArweaveMetaBundler) createMetaDataItem(payload *Payload, arweaveBloc
 	}
```

Comment:
each L1 interaction should be wrapped in a separate meta data item

Reply:
Why? The outcome in GQL is the same if we use a nested bundle